.eslintrc.js

```diff
@@ -356,6 +356,16 @@ module.exports = {
                 ],
                 allowSameFolder: true,
               },
+              {
+                target: ['src/core/**/*'],
+                from: [
+                  'plugins/**/*',
+                  'src/plugins/**/*',
+                  'src/legacy/core_plugins/**/*',
+                  'src/legacy/ui/**/*',
```

Comment:
Should we add x-pack for good measure?

Reply:
ah, it has been merged. ok, I will update PR tomorrow and ping the team